Makefile

```diff
@@ -306,7 +307,7 @@ patch-crds: bin/$(PLATFORM)/yq ## Patch-crds
 	hack/patch-crds.sh
 
 .PHONY: lint
-lint: bin/$(PLATFORM)/golangci-lint fmt vet ## Lint
```

Comment:
I believe we should not include the fmt target in the lint target. Consider the following two cases:

1. A user makes local commits and runs the lint target for a final check. Even if there are lint failures, the fmt target fixes them automatically, causing the lint target to succeed despite local diffs. As a result, the user may push the commit without realizing there are uncommitted changes.
2. In CI, make lint runs, and even if there are lint failures, the fmt target fixes them. This could cause the CI job to pass mistakenly, despite the issues.